cypress/e2e/usecase_page.cy.ts

```diff
@@ -0,0 +1,129 @@
+const API_URL = Cypress.env('apiUrl')
```

Comment:
I am not knowledgeable on cypress so I will just review the code and not the workings.